src/Combinatorics/SimplicialComplexes.jl

```diff
@@ -0,0 +1,353 @@
+using Markdown
+import Oscar: Polymake, pm_object
+
+export
+    SimplicialComplex,
+    betti_numbers,
+    dim,
+    euler_characteristic,
+    facets,
+    f_vector,
+    h_vector,
+    minimal_nonfaces,
+    nvertices,
+    stanley_reisner_ideal,
+    stanley_reisner_ring,
+    load_simplicialcomplex,
+    save_simplicialcomplex,
+    complex_projective_plane,
+    real_projective_plane,
+    klein_bottle,
+    torus # requires a distinction from, e.g., an algebraic group
+
+################################################################################
+##  Constructing
+################################################################################
+
+struct SimplicialComplex
+    pm_simplicialcomplex::Polymake.BigObject
+end
+
+pm_object(K::SimplicialComplex) = K.pm_simplicialcomplex
+
+
+@doc Markdown.doc"""
+    SimplicialComplex(generators::Vector{Vector{Int}})
+
+Construct an abstract simplicial complex from a set of faces.
+While arbitrary nonnegative integers are allowed as vertices, they will be relabeled to consecutive integers starting at 1.
+
+# Example
+```jldoctest
+julia> K = SimplicialComplex([[1,2,3],[2,3,4]])
+Abstract simplicial complex of dimension 2 on 4 vertices
+```
+# Simplicial complex comprising the empty set only
+```jldoctest
+julia> empty = SimplicialComplex(Vector{Set{Int}}([]))
+Abstract simplicial complex of dimension -1 on 0 vertices
+```
+# Example with relabeling
+The original vertices can be recovered.
+```jldoctest
+julia> L = SimplicialComplex([[0,2,17],[2,17,90]]);
+
+julia> facets(L)
+2-element Vector{Vector{Int64}}:
+ [1, 3, 2]
+ [3, 4, 2]
+
+julia> Oscar._vertexindices(Oscar.pm_object(L))
```

Comment:
We should not include internal examples in the docstring. If this information is to be accessible for the user, you might want to define something like `vertexindices(L::SimplicialComplex) = _vertexindices(pm_object(L))` and export that.